### PR TITLE
Skycoin wallet v0.20.2 released.de.md

### PR DESCRIPTION
**Pull request by Surf-N-Code**
DE translation, ready for review.

n1CJd4obWucqJvXtajAq52WYQ3ni44G8A8

### CODE REVIEW

File: content/statement/Skycoin Wallet v0.20.2 Released.de.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+bounty = 2
+date = "2017-10-13"
+categories = [
+    "Statement",
+]
++++
+
+[Download Skycoin Wallet v0.20.2](https://www.skycoin.net/downloads/)
+
+Dieses Release behebt viele Probleme der vorherigen Wallet Versionen und hat zudem eine neue Benutzeroberfläche.
```
Comment (IamVanCleef) [inline content/statement/Skycoin Wallet v0.20.2 Released.de.md]: Wallet-Versionen

File: content/statement/Skycoin Wallet v0.20.2 Released.de.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Das Upgrade der Benuzteroberfläche ist eine Verbesserung gegenüber v0.19, ist allerdings als Platzhalter anzusehen bis die Versionen v.021 oder v0.22 herauskommen.
```
Comment (IamVanCleef) [inline content/statement/Skycoin Wallet v0.20.2 Released.de.md]: herauskommen => erscheinen

File: content/statement/Skycoin Wallet v0.20.2 Released.de.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Die wichtigsten Änderungen auf einen Blick:
+
+- Zeigt immer die Korrekte Anzahl an Skycoins im Wallet.
```
Comment (IamVanCleef) [inline content/statement/Skycoin Wallet v0.20.2 Released.de.md]: korrekte Anzahl

File: content/statement/Skycoin Wallet v0.20.2 Released.de.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+- Behebt das Problem beschädigter Blockchain Cache Dateien.
+- Keine Performanceprobleme mehr. Die Benuzteroberfläche ist responsive.
```
Comment (IamVanCleef) [inline content/statement/Skycoin Wallet v0.20.2 Released.de.md]: Performance-Probleme oder Performanzprobleme

**Comment by IamVanCleef** [comment]
Is anyone working at this? If yes, it would be nice if the changes could be done so that the file can be merged. if not, then I am looking forward to translate some stuff...

**Comment by C3POv7** [comment]
me too ;)

**Comment by IamVanCleef** [comment]
could you please start working on it?

**Comment by Surf-N-Code** [comment]
Changes implemented.

**Pull request merged**